cabot/metricsapp/api/grafana.py

```diff
@@ -0,0 +1,195 @@
+import json
+import logging
+import requests
+import urlparse
+from django.core.exceptions import ValidationError
+
+
+logger = logging.getLogger(__name__)
+
+
+def _get_time_range(timestring):
```

Comment:
somebody wrote up a quick module for this - https://github.com/wroberts/pytimeparse

Not mandatory to use it. Just FYI